homeassistant/components/heos/media_player.py

```diff
@@ -311,7 +319,7 @@ def supported_features(self) -> int:
     @property
     def unique_id(self) -> str:
         """Return a unique ID."""
-        return str(self._player.player_id)
+        return self._device_id + "_player"
```

Comment:
Use new style string formatting instead of string concatenation.

Reply:
Why do we need to add `_player`?

Reply:
HEOS groups use the same ID as the lead player, so we when we add group support, we'll need to add a suffix to distinguish if it's a group entity or a player entity.  We may also add sensors in the future on the player device.  This is an attempt to avoid a potential future breaking change.